nexus/types/src/external_api/params.rs

```diff
@@ -1228,6 +1228,14 @@ pub struct InstanceCreate {
     /// `null`.
     #[serde(default)]
     pub auto_restart_policy: Option<InstanceAutoRestartPolicy>,
+
+    /// Affinity groups which this instance should be added into.
+    #[serde(default)]
+    pub affinity_groups: Vec<NameOrId>,
```

Comment:
If affinity groups are considered experimental, should we leave them out here at first?

Reply:
Yeah, good idea. I'll just delete this portion of the code for now.